src/components/taskDetails/index.tsx

```diff
@@ -352,6 +352,12 @@ const TaskDetails: FC<Props> = ({ taskID }) => {
                                             data-testid="request-task-button"
                                             className={classNames.button}
                                             onClick={taskRequestHandle}
+                                            disabled={
+                                                !Object.is(
+                                                    taskDetailsData?.assignee,
+                                                    null
```

Comment:
```suggestion
                                                !Object.is(
                                                    taskDetailsData?.assignee,
                                                    null
```
`const isTaskDeatilsData =  !Object.is(taskDetailsData?.assignee,null)`
and then use this variable as disabled value